pages/dkp/kaptain/1.2.0/tutorials/fairing/index.md

```diff
@@ -0,0 +1,314 @@
+---
+layout: layout.pug
+navigationTitle: Kubeflow Fairing
+title: Kubeflow Fairing
+menuWeight: 19
+excerpt: Tutorial for Kubeflow Fairing
+beta: false
+enterprise: false
+---
+
+<p class="message--note"><strong>NOTE: </strong>All tutorials in Jupyter Notebook format are available for
+<a href="https://downloads.mesosphere.io/kaptain/d2iq-tutorials-1.2.0.tar.gz">download</a>. You can either
+download them to a local computer and upload to the running Jupyter Notebook or run
+<code>curl -L https://downloads.mesosphere.io/kaptain/d2iq-tutorials-1.2.0.tar.gz | tar xz</code>
+from a Jupyter Notebook Terminal running in your Kaptain installation.
+</p>
+<p class="message--note"><strong>NOTE: </strong>These notebook tutorials have been built for and
+tested on D2iQ's Kaptain. Without the requisite Kubernetes operators and custom Docker images, these notebooks
+will likely not work.</p>
+
+
+<p class="message--warning"><strong>WARNING: </strong>Kubeflow Fairing does not support docker registries using
+a self-signed TLS certificate, certificate chaining nor insecure (plaintext HTTP) registries.</p>
+
+# Kubeflow Fairing: Build Docker Images from within Jupyter Notebooks
+
+## Introduction
+Although you can build Docker images by downloading files to your local machine and subsequently pushing the images to a container registry, it is much faster to do so without leaving Jupyter!
+[Kubeflow Fairing](https://www.kubeflow.org/docs/fairing/fairing-overview/) makes that possible.
+
+### What You Will Learn
+In this notebook you will go through the steps involved in building a Docker image from a base image (e.g. TensorFlow or PyTorch) and a custom trainer file that defines your machine learning model.
+This image can be used for distributed training or [hyperparameter tuning](../katib).
+You can use the model code you generated with `%%writefile` in [MNIST with TensorFlow tutorial](../training/tensorflow) or [MNIST with PyTorch tutorial](../training/pytorch) or a file of your own choosing.
+
+The Docker image builder process stores (temporary) files in MinIO.
+[MinIO](https://min.io/), an open-source S3-compliant object storage tool, is already included with your Kubeflow installation.
+
+### What You Need
+
+- An executable Python file (e.g. an `mnist.py` trainer - you can extract it from [Pytorch tutorial](../training/pytorch) in case you do not have one handy);
```

Comment:
should that be a period instead of a semicolon at the end?

Reply:
ouch. yes, likely. I'd suggest skipping this one as (a) this file is generated from Jupyter Notebook and a proper fix will require rebuild and tutorials artifact re-release and (b) we plan to remove fairing tutorials anyway. Maybe we can fix this in the docs PR only.